referrer-policy/no-referrer-policy/no-referrer-policy.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>Check that subresource gets the referrer URL when no explicit
+           Referrer Policy is set.</title>
+    <script src='/resources/testharness.js'></script>
+    <script src='/resources/testharnessreport.js'></script>
+    <!-- Common global functions for referrer-policy tests. -->
+    <script src="../generic/common.js"></script>
+  </head>
+  <body>
+    <h1>Check that subresource gets the referrer URL when no explicit Referrer
+        Policy is set.</h1>
+
+    <p>This page loads the subresource and waits for the message containing the
+       referrer URL from the child.</p>
+
+    <script>
+
+      // Display this page's location.
+      appendPageInfo("Parent frame");
+
+      // Embed the iframe on a different domain - load subresource.
+      var iframe = document.createElement('iframe');
+      iframe.type = 'text/javascript';
+      iframe.src = "http://www1." + location.hostname + ":" + location.port +
+                   "/referrer-policy/no-referrer-policy" +
+                   "/no-referrer-policy.subresource.html";
+      iframe.width="100%";
+      iframe.height="200px";
+      document.body.appendChild(iframe);
+
+      var referrerTest = async_test(
+          "Checks that referer is passed to the subresource (iframe).");
+
+      function iFrameListener(event) {
+        referrerTest.step(function() {
+            // Sanity check - location of the iframe matches reported location.
+            assert_equals(event.data.location, iframe.src,
+                          "Check child reported location.");
+
+            // Child reports the referrer URL.
+            assert_equals(event.data.referrer, document.location.toString(),
```

Comment:
This isn't quite right, at least not in the general case. We strip fragment identifiers from referrers, which we should eventually have a test for. It might be reasonable to add a `stripForUseAsReferrer` function to your utility JavaScript file which implements https://w3c.github.io/webappsec/specs/referrer-policy/#strip-url for this comparison.